tests/storage/linstor/test_linstor_sr.py

```diff
@@ -52,6 +57,29 @@ def test_create_and_destroy_sr(self, pool_with_linstor, provisioning_type, stora
         vm.destroy(verify=True)
         sr.destroy(verify=True)
 
+
+def get_drbd_status(host: Host, resource: str):
+    logging.debug(f"[{host}] Fetching DRBD status for resource `{resource}`...")
+    return json.loads(host.ssh(["drbdsetup", "status", resource, "--json"]))
+
+def get_corrupted_resources(host: Host, resource: str):
+    return [
+        (
+            res.get("name", ""),
+            conn.get("name", ""),
+            peer.get("out-of-sync", 0),
+        )
+        for res in get_drbd_status(host, resource)
+        for conn in res.get("connections", [])
+        for peer in conn.get("peer_devices", [])
+        if peer.get("out-of-sync", 0) > 0
+    ]
+
+def wait_drbd_sync(host: Host, resource: str):
+    logging.info(f"[{host}] Waiting for DRBD sync on resource `{resource}`...")
```

Comment:
This works but pylint dont like fstrings: W1203: Use lazy % formatting in logging functions (logging-fstring-interpolation)